addons/account/static/src/components/product_label_section_and_note_field/product_label_section_and_note_field.js

```diff
@@ -162,7 +162,11 @@ export class ProductLabelSectionAndNoteField extends Many2OneField {
     }
 
     get isProductClickable() {
-        return this.props.record.model.root.data.state !== "draft";
+        if (this.props.record.model.root.data) {
+            return this.props.record.model.root.data.state !== "draft";
+        }
+        
+        return this.props.record._parentRecord.data.state !== "draft";
```

Comment:
General remark: you're not supposed to access private properties (prefixed by _). I don't know what the issue/fix are though. If it's related to studio, @kebeclibre would know